projects/fsastorefrontlib/src/occ/adapters/consent-management/occ-consent.adapter.ts

```diff
@@ -62,15 +66,27 @@ export class OccConsentAdapter implements ConsentAdapter {
       .pipe(catchError((error: any) => throwError(error.json())));
   }
 
-  getOBOCustomerList(userId: string): Observable<any> {
+  getOBOCustomerList(
+    userId: string,
+    searchConfig?: FSSearchConfig
+  ): Observable<OBOCustomerList> {
     const url = this.occEndpointService.buildUrl('oboConsentCustomers', {
       urlParams: {
         userId,
       },
     });
-    const params = new HttpParams({ fromString: FULL_PARAMS });
+    let params = new HttpParams({ fromString: FULL_PARAMS });
+    if (searchConfig?.sortCode && searchConfig?.sortOrder) {
+      params = params
+        .set('sortCode', searchConfig.sortCode)
+        .set('sortOrder', searchConfig.sortOrder);
```

Comment:
If I'm reading this right this means that the first parametar that is being set (`sortCode`) will actually never get passed, instead the only `sortOrder` will be passed? Or am I missing something? This is what is also confirmed on that link you've [posted](https://stackoverflow.com/questions/51980630/what-is-the-difference-between-the-set-and-append-methods-of-angulars-httpparams#answer-51981150).

Reply:
No, it won't be the case. Both parameters will be set, but if you call one more set method to set for example `sortCode` parameter, it will override the first value and set a new one for that parameter.

Reply:
Yes, but that was my point the whole time. If user selects to have descending sort order and sort by status for instance, then one of those two wont be sent to the BE, right?

Reply:
Everything will be sent to the BE! If it works like you predict we will have problems throughout entire the application. Go again through examples or just try on your own on some example in the app.